index.html

```diff
@@ -4975,6 +4975,621 @@ <h2 id="AudioWorkerParamDescriptor">
+            script code is evaluated in this scope to define
+            <a>AudioWorkletProcessor</a>.
+          </p>
+          <dl title="interface AudioWorkletGlobalScope : WorkletGlobalScope" 
+          class="idl">
+            <dt>
+              void registerAudioWorkletProcessor (DOMString name, Function classConstructor)
+            </dt>
+            <dd>
+              <p>
+                Registers a definition of a class that is derived from
+                <a>AudioWorkletProcessor</a>.
+              </p>
+              <dl class="parameters">
+                <dt>
+                  DOMString name
+                </dt>
+                <dd>
+                  A string key that represents a class definition to be
+                  registered. <strong>ISSUE</strong>: The conflict between
+                  multiple class definitions under the same name needs to be
+                  addressed. For the time being, WG’s resolution is to apply the
+                  first-come-first- served policy.
+                </dd>
+                <dt>
+                  Function classConstructor
+                </dt>
+                <dd>
+                  A class definition extended from <a>AudioWorkletProcessor</a>.
+                </dd>
+              </dl>
+            </dd>
+          </dl>
+        </section>
+        <section>
+          <h2 id="AudioWorkletProcessor">
+            The <dfn>AudioWorkletProcessor</dfn> interface
+          </h2>
+          <p>
+            This interface represents an audio processing module that runs on
+            the audio <a>rendering thread</a>. It is defined in an
+            <a>AudioWorkletGlobalScope</a> and the definition of such class
+            manifests the actual audio processing mechanism of an
+            <a>AudioWorkletNode</a>. <a>AudioWorkletProcessor</a> can only be
+            instantiated by the construction of an <a>AudioWorkletNode</a>
+            instance.
+          </p>
+          <dl title="interface AudioWorkletProcessor" class="idl">
+            <dt>
+              readonly attribute static sequence&lt;AudioParamDescriptor&gt;
+            </dt>
+            <dd>
+              <p>
+                This array contains <a>AudioParamDescriptors</a>, which are
+                descriptors for each controllable parameter on an
+                <a>AudioNode</a>. This static property is referred in the
+                construction of an <a>AudioWorkletNode</a> to create instances
+                of <a>AudioParam</a> in the node.
+              </p>
+            </dd>
+            <dt>
+              readonly attribute AudioContextInfo contextInfo
+            </dt>
+            <dd>
+              <p>
+                This getter provides an object contains the information of the
+                associated <a>AudioContext</a> such as the sample rate and the
+                current playback time when queried.  
```

Comment:
Since AudioContextInfo object is described somewhere else, don't really need to describe here what it might contain.  Leave this out.